src/instaskip/actions/routes.clj

```diff
@@ -0,0 +1,12 @@
+(in-ns 'instaskip.actions)
+
+(defn list-routes [filters innkeeper-config]
+
+  (match filters
+         {:team team}
+         ; =>
+         (ik/get-paths {:owned-by-team team} innkeeper-config)
+
+         :else
+         ; =>
```

Comment:
I stole it from this [article](http://matt.might.net/articles/parsing-with-derivatives/). I guess the intention is to make the code more readable for newbies. Do you think I should remove it? Or do you think it helps? :)